boltdesignsystem/bolt
# DS-504: Add option to show and hide Test folder in Pattern Lab

## Jira

https://pegadigitalit.atlassian.net/browse/DS-504

## Summary

As mentioned in the ticket I provide the way to show and hide the `Tests` folder.

## Details

The solution that I was provided is different from the proposal in the ticket. From my perspective is better because if someone wants to have `Tests` visible, he will have this opportunity right now. Please let me know if you this that we shouldn't use this approach.

![image](https://user-images.githubusercontent.com/16049049/126997980-cbd42fb0-8fa5-406f-b2bb-4e9bcfd30279.png)

![image](https://user-images.githubusercontent.com/16049049/126998094-491b5d06-6ff4-48e1-b560-138dc595c419.png)

## How to test

1. Run this code locally or use this link https://feature-ds-504-test-folder-flag.boltdesignsystem.com
2. Go to component or element view or whatever view that you can use Pattern Lab options.
3. Click on `Show Test Folder`
4. `Tests` folder should appear below `Experiments`
5. Click on `Hide Test Folder` 
6. `Tests` folder should disappear from the menu.
